airmozilla/manage/views.py

```diff
@@ -3643,7 +3647,8 @@ def _get_all_pictures(event=None):
             'height': picture.height,
             'size': picture.size,
             'created': picture.created.isoformat(),
-            'events': event_map[picture.id]
+            'events': event_map[picture.id],
+            'event': picture.event
```

Comment:
Change this to `'event': picture.event_id` instead. That way it doesn't make another SQL query just to materialize that into an ORM object.